### PR TITLE
feat(authoring): SD-3471 Added user avatar in top header of locked items

### PR DESCRIPTION
**Pull request by pavlovicnemanja**
SD-3471

### CODE REVIEW

File: client/app/scripts/superdesk-authoring/views/authoring-topbar.html [hunk 1]
```diff
@@ -1,16 +1,38 @@
...
+            <div class="user-image">
+                <img sd-user-avatar data-src="item.lock_user.picture_url" data-initials="item.lock_user | username">
+            </div>
+        </figure>
+
+        <div class="dropdown-menu">
+            <div class="menu-label" translate>Locked by</div>
+            <figure class="avatar large">
+                <div class="user-image">
+                    <img sd-user-avatar data-src="item.lock_user.picture_url" data-initials="item.lock_user | username">
```
Comment (plamut) [inline client/app/scripts/superdesk-authoring/views/authoring-topbar.html]: The `` tag should be closed, right?

Comment (pavlovicnemanja) [inline client/app/scripts/superdesk-authoring/views/authoring-topbar.html]: By W3, img tag doesn't need to be closed at the end: http://www.w3schools.com/tags/tag_img.asp

**Comment by plamut** [comment]
IMO a test should be added for changes in the `sdUserAvatar` directive, but looks good to me otherwise.

File: client/app/scripts/superdesk-authoring/views/authoring-topbar.html (hunk 1 above)
Comment (plamut) [inline client/app/scripts/superdesk-authoring/views/authoring-topbar.html]: Aha, I've re-checked and you're right, this is not the case anymore in HTML5.

**Comment by ioanpocol** [comment]
:+1:

**Pull request merged**